tasks/checkers.md

```diff
@@ -0,0 +1,31 @@
+# Summary
+> Groups play checkers against a computer
+
+# References
+> Woolley, A. W., Chabris, C. F., Pentland, A., Hashmi, N., & Malone, T. W. (2010). Evidence for a Collective Intelligence Factor in the Performance of Human Groups. Science, 330(6004), 686 LP – 688. https://doi.org/10.1126/science.1193147
+> Supplementary information: https://science.sciencemag.org/content/sci/suppl/2010/09/29/science.1193147.DC1/Woolley_SOM_Revision_1.pdf
+
+# Procedure
+## Steps
+### Pre-survey
+>1. Participants completed Raven’s Advanced Progressive Matrices, a standardized test of general fluid reasoning. 
+
+### Task
+> 1. Group members were seated in front a computer and told they would play checkers against a computer
+> 2. Members familarized themselves with the game and rules individually
+> 3. Groups had 5 minutes to discuss strategy
+> 4. Teams played a practice match against each other
+> 5. Teams played one test match against the computer
+
+## Instructions
```

Comment:
I think this section is supposed to be the exact instructions given to participants like in this example: https://docs.google.com/document/d/1yxQfHobL3vAamNgnt14korvORaI-N1ed62gHsEovHXI/edit

I feel like the ones outlined here align/could be integrated into the procedure